modules/cpmso-yaml-provider-spec-vsphere.adoc

```diff
@@ -50,4 +50,5 @@ providerSpec:
 <11> Specifies the vCenter Datastore for the control plane.
 <12> Specifies the path to the vSphere VM folder in vCenter, such as `/dc1/vm/user-inst-5ddjd`.
 <13> Specifies the vSphere resource pool for your VMs.
-<14> Specifies the vCenter server IP or fully qualified domain name.
\ No newline at end of file
+<14> Specifies the vCenter server IP or fully qualified domain name.
+//Check for 4.15 FD behavior wrt subnet (not applicable?)
```

Comment:
Do you have a Jira item handy for this implementation?

Reply:
thanks! will look into that. Pushing other changes for now.

Reply:
Ok, looks like another writer covered this one. I will remove that note and then I think this can be sent for QE. Let me know if you think I have missed anything in our conversation on this PR :)